backtester/data/data.go

```diff
@@ -7,14 +7,15 @@ import (
 
 	"github.com/thrasher-corp/gocryptotrader/backtester/common"
 	gctcommon "github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/common/key"
 	"github.com/thrasher-corp/gocryptotrader/currency"
 	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
 )
 
 // NewHandlerHolder returns a new HandlerHolder
 func NewHandlerHolder() *HandlerHolder {
 	return &HandlerHolder{
-		data: make(map[string]map[asset.Item]map[*currency.Item]map[*currency.Item]Handler),
+		data: make(map[key.ExchangePairAssetKey]Handler),
```

Comment:
```suggestion
		data: make(map[key.ExchangePairAssetKey]Handler),
```

As discussed, I think there's enough context with the package import to know that the exported type is a key so we can prevent stutter